pkg/network/ovn_kubernetes.go

```diff
@@ -256,48 +248,15 @@ func networkPluginName() string {
 	return "ovn-kubernetes"
 }
 
-type replicaCountDecoder struct {
-	ControlPlane struct {
-		Replicas string `json:"replicas"`
-	} `json:"controlPlane"`
-}
-
 func bootstrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
```

Comment:
If we're not doing anything special at install time any more, we shouldn't need any "bootstrap" code at all; we should just be able to do it all from `renderOVNKubernetes` without it needing a `bootstrapResult`

Reply:
Not quite. We still need to bootstrap and provide OVN (nbdb/sbdb/northd/ovnkube) with the entire list of master IPs when deploying the stack. Once upon a time I tried to remove that (#447) and failed miserably

Reply:
But how can you provide the entire list of master IPs if you don't wait for all the masters to be created?

Reply:
If the dbs, etc, actually need the _complete_ list of masters when they are first configured, then this PR is wrong. If they just need a list of _some_ masters and we can add to that list later and everything will keep working, then you don't need bootstrapping code, you just need `renderOVNKubernetes` to re-fetch the list of master IPs when the set of masters changes.

Reply:
>  If they just need a list of some masters and we can add to that list later and everything will keep working

That's it. 

> then you don't need bootstrapping code, you just need renderOVNKubernetes to re-fetch the list of master IPs when the set of masters changes.

Does it matter though? `renderOVNKubernetes` would need to do exactly what `bootstrapOVN` does today, i.e: get all master nodes's internal IP and render the template with it. Or am I overlooking something that would simply things further?

Reply:
> Does it matter though?

Architecturally, there is supposed to be a distinction between bootstrap and non-bootstrap, though I don't know exactly what it is since I guess we re-run the bootstrap code even after install time? @squeed should know...

Reply:
Bootstrap was initially designed for Kuryr, which needs to create certain cloud-provider resources on install. Given that we also wanted to support the offline renderer, it seemed like a useful design.

It turns out we didn't need to render offline, so the distinction between "bootstrap" and "render" phases is no longer necessary. However, it does make unit testing a **lot easier**, so I'd like to keep it if possible.

I have a suggested design, which I'll add in a comment below, since it isn't really a line-level suggestion.